cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -388,6 +388,44 @@ func TestGetAPIServerCommand(t *testing.T) {
 				"--etcd-keyfile=faz",
 			},
 		},
+		 {
+                        cfg: &kubeadmapi.MasterConfiguration{
```

Comment:
Please copy the most simple case there is here; and set CloudProvider. Then, the PersistentVolumeLabel value should be in the list.